src/Sarif.Multitool.Library/Rules/RuleId.cs

```diff
@@ -49,6 +49,17 @@ public static class RuleId
         public const string ProvideCheckoutPath = "GH1006";
         public const string ProvideFullyFormattedMessageStrings = "GH1007";
 
+        // Rules for GHAS.
+        public const string GHASReferenceFinalSchema = "GH1011";
```

Comment:
>GH1011 [](http://example.com/codeflow?start=56&length=6)

changed `GHAS1011` to `GH1011` 

I moved up here so it can be close to the existing GH rules.